src/Credit/Credit.cpp

```diff
@@ -28,18 +28,34 @@ struct CreditWidget : ModuleWidget {
     ModuleWriter mWriter;
 
     static void saveModules(ModuleWriter& mWriter) {
-        auto pathC = file_utils::getChosenFilePath();
+#ifdef USING_CARDINAL_NOT_RACK
+        std::string dir, ignore;
+        file_utils::getDefaultFilePath(dir, ignore);
+        async_dialog_filebrowser(true, dir.c_str(), "Save credit file", [&mWriter](char* pathC) {
```

Comment:
I'm guessing async_dialog_filebrowser is a method only available through the Cardinal API? It would be nice to leave a comment saying something to that effect, just for future reference.

Reply:
sure, though I think the macro kinda gives that away